index.html

```diff
@@ -1,38 +1,22 @@
+<!-- Type '!' and press enter for the following information in the <head> tag to be processed-->
+<!-- This information is so that the page (images, text, etc) is responsive when you change the size of the page-->
+<!-- Also I changed the license to fit your changed username (I changed "Copyright (c) 2022 omardadeveloper" to "Copyright (c) 2022 omarshanid"), don't know if it that important though.-->
```

Comment:
actually this is helpful thx would have never noticed unless you pointed out

Reply:
np